docs/code-quality/ca2248.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'CA2248: Provide correct 'enum' argument to 'Enum.HasFlag''
+ms.date: 05/06/2020
+ms.topic: reference
+f1_keywords:
+- CA2248
+helpviewer_keywords:
+- CA2248
+author: evangelink
+ms.author: evangelink
+manager: 
```

Comment:
Not sure how you fill these fields.

Reply:
We need an internal MS name for these. I'll add the suggested edits.